ui/lib/core/addon/components/edit-form.js

```diff
@@ -20,8 +20,6 @@ export default Component.extend({
   modelValidations: null,
   // public API
   model: null,
-  successMessage: 'Saved!',
```

Comment:
May want to do a global search for this, since IIRC lots of things extend `edit-form`